index.html

```diff
@@ -49,10 +76,209 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+                              A dayly selection of privatelypersonalized reads:no accounts <br>
+                              or sign-ups required.
+                          </p>
+                      </div>
+                      <div >
+                          <ul class="languages">
+                          <li class="language">html</li>
+                          <li class="language">css</li>
+                          <li class="language">javascript</li>
+                          </ul>
+                      </div>
+                      <div><button class="see-project" type="button">See project</button></div>
+                  </div>
+                    </div>
+          
+          
+                    <div class="Works">
+                      <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot4.png" alt=""></div>
+              
+              
+                      <div class="tonic">
+                          <div><h1>Muti-Post <br>Stories</h1></div>
+      
+                          <div class="flex1">
+                              <div>Caponi</div>
+                              <div class="round"></div>
+                              <div>back End</div>
+                              <div class="round"></div>
+                              <div>2015</div>
+                          </div>
+      
+                          <div>
+                              <p>
+                                  A dayly selection of privatelypersonalized reads:no accounts <br>
+                                  or sign-ups required.
+                              </p>
+                          </div>
+      
+                          <div >
+                              <ul class="languages">
+                              <li class="language">html</li>
+                              <li class="language">css</li>
+                              <li class="language">javascript</li>
+                              </ul>
+                          </div>
+      
+                          <div ><button class="see-project" type="button">See project</button></div>
+                      </div>
+                        </div>
+                   
+          
+          
+           <div class="about-me" >
+          
+             <div class="first-about">
+              <div >
+                  <h1>About<br>Myself</h1>
+              </div>
+          
+      
+      
+              
+          <div class="p-div" >
+              <p class="line">I’m a software developer! I can help you build a product , feature or website Look through <br> 
+                  some of my work and experience! If you like what you see and have a project you need <br>
+                  coded, don’t hestiate to contact me</p>
+          </div>
+          
+                  <div class="div-CONNECT2" >
+                      <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
+                  </div>
```

Comment:
Good job updating the styling of the about me section. However, a few styling issues still need to be met to match the figma design better 🙏.

Your design:
<img width="238" alt="image" src="https://user-images.githubusercontent.com/37116322/220424125-dc2ef6ce-b69c-4eb0-b449-1767a2b2123d.png">

Figma design:
<img width="228" alt="image" src="https://user-images.githubusercontent.com/37116322/219808404-f9824f4f-7bb7-40b2-bedf-1b017edc2d06.png">

A few things to note:
- The font size of the about myself title should be bigger
- The social icons should be slightly bigger and aligned to the left
- The get my resume button should have the same styling as the see project button and also have button interactiosn styled

**Please try your best to make sure that your design matches the Figma design as much as possible😎**